mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -70,7 +70,7 @@ public static string MapTimeZoneId (string id)
 				case "Europe/Rome":
 				case "Europe/Vatican":
 					return "W. Europe Standard Time";
-				case "Canada/Eastern":
+				case "America/Toronto":
```

Comment:
`TimeZoneTest.FindSystemTimeZoneById()` needs to be changed to `America/Toronto` too.